luis-eduardo-salcedo/Tarea-11/router/src/components/navbar.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from "react-router-dom";
+
+export default function navBar(){
+    return(
+        <>
+            <ul>
+                <li><Link className="nav active" to="/">Home</Link></li>
```

Comment:
@luisSalcedo8 , this route list should be dynamic and build from an Array using map functions